sound/soc/sof/sof-audio.h

```diff
@@ -63,7 +63,9 @@ struct snd_sof_audio_ops {
 
 	/* DAI ops */
 	struct snd_soc_dai_driver *drv;
-	int num_drv;
+	int num_ssp_drv;
+	int num_hda_drv;
+	int num_dmic_drv;
 };
```

Comment:
```struct snd_sof_audio_ops``` should be generic.

Here we should create a better definition for DAIs. Maybe we add an array where each element contains the name of the dai and the number.

Reply:
I've moved this to the intel chip info. I guess that's reasonable right?